### PR TITLE
Normative: Splat out the property descriptor in the decorator descriptor

### PR DESCRIPTION
**Pull request by littledan**
This is an ergonomics improvement, as discussed in #173

### CODE REVIEW

File: METAPROGRAMMING.md [hunk 1]
```diff
@@ -189,11 +191,10 @@ function bound(elementDescriptor) {
   // Return both the original method and a bound function field that calls the method.
```
Comment (nicolo-ribaudo) [inline METAPROGRAMMING.md:190]: (A few lines above, GitHub doesn't let me to comment there).
The destructuring needs to be updated to remove `descriptor`.

File: METAPROGRAMMING.md [hunk 2]
```diff
@@ -206,22 +207,19 @@ function observed({kind, key, placement, descriptor, initializer}) {
   assert(placement == "own");
```
Comment (nicolo-ribaudo) [inline METAPROGRAMMING.md:206]: Also 2 lines above here

File: METAPROGRAMMING.md (hunk 1 above)
Comment (littledan) [reply METAPROGRAMMING.md:190]: Good catch, fixed

File: METAPROGRAMMING.md (hunk 2 above)
Comment (littledan) [reply METAPROGRAMMING.md:206]: fixed

**Comment by littledan** [comment]
OK, merging this patch per previous discussion, but I still welcome comments.

**Pull request merged**